Makefile

```diff
@@ -24,7 +24,7 @@ BUILDX_FLAGS=$(BUILDX_OP) $(PLATFORM_FLAGS) $(CACHE_FLAGS)
 # ensuring all user crates are compiled with the same rustc toolchain
 RUSTUP_TOOLCHAIN=1.70.0
 
-TAG?=$(shell git describe --tags)
+TAG?=$(shell git describe --tags --abbrev=0)
```

Comment:
I can remove this from this PR, but I don't see any problem with it, and it means less rebuilding from scratch since we don't get a new tag on every commit.